### PR TITLE
Skeleton task

### PR DESCRIPTION
**Pull request by Ascarbek**
This PR is intended to prepare "AppFlowy Tauri" project for development.

Features:
 - code format using Prettier
 - code style check using ESLint 
 - setup Vite to serve assets from another folder (app_flowy flutter)
 - import Poppins font
 - setup tailwind colors to use palette from Figma UI Kit
 - install redux/toolkit for typed states and typed action dispatch
 - basic layout components
 - Navigation component UI using special react hook

### CODE REVIEW

File: frontend/appflowy_tauri/src/appflowy_app/components/user/application/notifications/user_listener.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ export class UserNotificationListener extends AFNotificationListener<UserNotific
   onUserSignIn?: OnUserSignIn;
 
   constructor(params: {
-    userId?: String;
+    userId?: string;
```
Comment (appflowy) [inline frontend/appflowy_tauri/src/appflowy_app/components/user/application/notifications/user_listener.ts:13]: What different between String and string?

File: frontend/appflowy_tauri/package.json [hunk 2]
```diff
@@ -7,24 +7,33 @@
     "dev": "vite",
     "build": "tsc && vite build",
     "preview": "vite preview",
-    "tauri": "tauri"
+    "tauri:dev": "tauri dev"
```
Comment (appflowy) [inline frontend/appflowy_tauri/package.json:10]: Also modify the `[tasks.tauri_dev]`

`npm run tauri dev` -> `npm run tauri:dev`

File: frontend/appflowy_tauri/src/appflowy_app/components/user/application/notifications/user_listener.ts (hunk 1 above)
Comment (Ascarbek) [reply frontend/appflowy_tauri/src/appflowy_app/components/user/application/notifications/user_listener.ts:13]: https://typescript-eslint.io/rules/ban-types/

The 'string' is more correct.

Comment (appflowy) [reply frontend/appflowy_tauri/src/appflowy_app/components/user/application/notifications/user_listener.ts:13]: Got it

**Pull request merged**